src/components/AssetHeader/AssetChart.tsx

```diff
@@ -77,6 +86,25 @@ export const AssetChart = ({ accountId, assetId, isLoaded }: AssetChartProps) =>
     selectCryptoHumanBalanceFilter(s, opportunitiesFilter),
   )
 
+  const renderPrice = useMemo(() => {
+    if (!marketData?.price)
```

Comment:
Since the whole market-data is nullable, no need to check for price (should be accidentally fine since it's a string and `Boolean('0') === true` but...)
```suggestion
    if (!marketData)
```